tests/test_unit/stub_data/test.xml

```diff
@@ -126,6 +126,7 @@
       <ack>
          <title>Acknowledgments</title>
          <p>WE ACKNOWLEDGE.</p></ack>
+      <p><italic>Facility:</italic> <named-content content-type="facility" xlink:href="FacilityName"><italic>FacilityName</italic></named-content>.</p>
```

Comment:
We don't have a unit test to verify this gets extracted, right?

Reply:
Unit test added